src/pages/Aggregator/Aggregator.fields.ts

```diff
@@ -8,8 +8,8 @@ export enum AggregatorInputs {
   SendAmount = 'sendAmount',
   DestinationChain = 'destinationChain',
   DestinationToken = 'destinationToken',
-  ReceiveAmount = 'receiveAmount',
   ReceiveAddress = 'receiveAddress',
+  ReceiveAmount = 'receiveAmount'
```

Comment:
You can leave it as-is now but this just produces unnecessary changes and makes PRs harder to review since there was no real change in this file.